common/src/commonMain/kotlin/com/trikot/sample/viewmodels/base/BaseNavigationDelegate.kt

```diff
@@ -0,0 +1,3 @@
+package com.trikot.sample.viewmodels.base
+
+interface BaseNavigationDelegate
```

Comment:
Nothing here to show what a navigation delegate should look like?

Reply:
I did not wanted to put too much opinion here. Remember scratch ?

Reply:
True!

Reply:
I would drop the `Base` prefix personnally